src/org/spdx/licenseTemplate/HtmlTemplateOutputHandler.java

```diff
@@ -98,17 +98,17 @@ private void removeEndParagraphTag() {
 	 */
 	public static String formatReplaceabledHTML(String text, String id) {
 		StringBuilder sb = new StringBuilder();
-		sb.append("\n<span ");
+		sb.append("\n<div ");
```

Comment:
This code is used for the old license spreadsheet and license text.  I'll change this back to spans since the old format will not include any phrasing content.